src/patients/hooks/useAddVisit.tsx

```diff
@@ -0,0 +1,41 @@
+import { isEmpty } from 'lodash'
+import { queryCache, useMutation } from 'react-query'
+
+import PatientRepository from '../../shared/db/PatientRepository'
+import Visit from '../../shared/model/Visit'
+import { uuid } from '../../shared/util/uuid'
+import validateVisit from '../util/validate-visit'
+
+interface AddVisitRequest {
+  patientId: string
+  visit: Omit<Visit, 'id' | 'createdAt'>
```

Comment:
we should export this type, probably from a different file, so that it can be used in the `AddVisitModal`